react/src/store/sagas/AuthSagas.js

```diff
@@ -6,14 +6,23 @@ import AuthService from '../../services/AuthService';
 
 export function* userLogin({ payload }) {
   try {
-    
-    yield call(AuthService.login, payload);
+    const user = yield call(AuthService.userData, payload.username);
 
-    yield put(authUser(true));
-    yield put(push('/home'));
-    yield put(go());
+    if(user[0].verified) {
+      yield call(AuthService.login, payload);
+
+      yield put(authUser(true));
```

Comment:
sto true? Sta je true?

Reply:
To je bilo u kodu, postavlja valjda pored tokena na sesiju i neku oznaku da je autentifikovan

Reply:
A ako si mislio na ovo verified, to mi je oznaka u bazi da je prosao verifikaciju na mejlu kako ne bi mogao da se uloguje dok ne prodje to